app/extensions/edm/__init__.py

```diff
@@ -190,11 +190,16 @@ def request_passthrough_parsed(
 
             message = {'unknown error'}
             error = None
+            vulnerable_sighting_guid = None
```

Comment:
This seems like the wrong way to do this. 
Response_data has fields in it and he fields will be specific to the request message sent. This code should not know nor care about request specific details in the response.
Instead, the HoustonException should pass the full response_data to the calling code. 
Then in this case the encounterfs code would know to extract the vulnerable sighting and individual.